src/common/harvest/harvest.js

```diff
@@ -97,24 +101,28 @@ export class Harvest extends SharedContext {
 
     if (!opts) opts = {}
 
-    var url = this.getScheme() + '://' + info.errorBeacon + '/' + endpoint + '/1/' + info.licenseKey + this.baseQueryString()
-    if (payload.qs) url += encodeObj(payload.qs, agentRuntime.maxBytes)
+    var url = customUrl || this.getScheme() + '://' + info.errorBeacon + '/' + endpoint + '/1/' + info.licenseKey
 
+    var baseParams = includeBaseParams ? this.baseQueryString() : ''
```

Comment:
doesn't `includeBaseParams` get passed down from harvest send as `undefined` forcibly so the default never applies?

Reply:
includeBaseParams uses the default (true) for every call except session replay at the moment